ansible_collections/arista/avd/molecule/eos_validate_state/inventory/host_vars/dc1-leaf2c.yml

```diff
@@ -0,0 +1,16 @@
+# TODO: Remove the example test of Vxlan1 in AVD 6.0.0 version
+custom_structured_configuration_vxlan_interface:
+  Vxlan1:
+    description: dc1-leaf2c_VTEP
+    vxlan:
+      udp_port: 4789
+      source_interface: Loopback1
```

Comment:
Is it okay to add this to existing host to avoid new catalog?

Reply:
Existing hosts are vtep devices and its generate vxlan1 from eos_designs and Vxlan1 from host vars. So our validation code will always check for vxlan1 first then Vxlan1 so wont able to test fully. Hence added in new host.